Modules/selectmodule.c

```diff
@@ -2148,37 +2148,31 @@ kqueue_queue_control(kqueue_queue_Object *self, PyObject *args)
     }
 
     if (ch != NULL && ch != Py_None) {
-        it = PyObject_GetIter(ch);
-        if (it == NULL) {
-            PyErr_SetString(PyExc_TypeError,
-                            "changelist is not iterable");
+        seq = PySequence_Fast(ch, "changelist is not iterable");
+        if (seq == NULL) {
             return NULL;
         }
-        nchanges = PyObject_Size(ch);
-        if (nchanges < 0) {
-            goto error;
-        }
+        nchanges = PySequence_Fast_GET_SIZE(arg);
```

Comment:
`PySequence_Fast_GET_SIZE` returns a Py_ssize_t. This may trigger a compile warning.